src/AutorestSwift/View Models/KeyValueViewModel.swift

```diff
@@ -28,10 +28,12 @@ import Foundation
 
 enum KeyValueDecodeStrategy: String {
     case dateFromParam
-    case byteArrayFromParam
-    case `default`
     case dateFromSignature
```

Comment:
Why no dataFromParam?

Reply:
There is dateFromParam, line 30. I just move down byteArrayFromParam to line 32.